### ISSUE TITLE
Add of the way of how will you manage global state 

### ISSUE THREAD

**Issue by cinfinit** [cinfinit/react-approaches-pratices#2]
Add one  of the way of how will you manage global state

**Comment by ThPadelis** [comment]
I could help with this one

**Comment by cinfinit** [comment]
Sure , but have you gone through the guidelines ? i mean for whom this repo is helpful for and all ?

**Comment by smooth-felix** [comment]
Hi! I would love to help with this.

**Comment by cinfinit** [comment]
Hey @smooth-felix , i hope you have gone through guidelines ? Let me know if you have gone through and want to start working on this

**Comment by smooth-felix** [comment]
@cinfinit Yes I went through the guidelines. I would like to start working on it.

**Comment by cinfinit** [comment]
Sure @smooth-felix  , assigning you this

**Comment by cinfinit** [comment]
Hey @smooth-felix , which approach are you working on ?

**Comment by smooth-felix** [comment]
@cinfinit  I am working on implementing a redux store with react-redux

**Comment by Yash-Yadav-git** [comment]
Is this issue still open?
I would like to contribute, my approaches would include 3 breakdowns,
1. Fetching data and managing globally using context API and useContext & useState hook.
2. Would include context API with reducer
3. Managing state globally using redux toolkit, store and slice apporach

**Comment by smooth-felix** [comment]
@cinfinit since routing is not managed in the application and that it requires to have a separate route for a demo app of the feature we are developing, should I first add the routing for the app and then continue on the global state handling?

**Comment by cinfinit** [comment]
Hi @Yash-Yadav-git , cool to hear on that , for the time-being as you can see in this thread @smooth-felix is working on the redux approach , so let's create a separate ticket for the approaches you have mentioned , let's revert on that ticket .

--Update 
@Yash-Yadav-git  created separate ticket , link below , please comment on that ticket so that i can assign you that.
https://github.com/cinfinit/react-approaches-practices/issues/4

**Comment by cinfinit** [comment]
Sure @smooth-felix , you can go ahead with routing and then global state handling

**Comment by cinfinit** [comment]
Hey @Yash-Yadav-git  , are you working on https://github.com/cinfinit/react-approaches-practices/issues/4 ? , If not ,  please let me know in this thread

**Comment by Yash-Yadav-git** [comment]
Hey @cinfinit I have raised an interest to assign the Issue to me.
Was away from my machine for few days, thus, the delay

**Comment by cinfinit** [comment]
Hey @smooth-felix  , what's the status for this ?

**Comment by smooth-felix** [comment]
Hi, @cinfinit today the redux store with example project + react routing is already added. I am currently working on the documentation. will make the PR tomorrow.

**Comment by cinfinit** [comment]
Hey @smooth-felix , any status/update on this ?
